firecracker-control/local.go

```diff
@@ -311,3 +343,93 @@ func (s *local) getVM(id string) (*instance, error) {
 
 	return instance, nil
 }
+
+// PatchStubDrive will replace a stub drive in the queue with a new path.
+func (s *local) PatchStubDrive(ctx context.Context, req *proto.PatchStubDriveRequest) (*proto.PatchStubDriveResponse, error) {
+	if s.stubDriveIndex >= len(s.drives) {
+		return nil, ErrStubDrivesExhausted
+	}
+
+	d := s.drives[s.stubDriveIndex]
```

Comment:
How do you feel about having `Drives map[string]string` (key=PathOnHost, value=DriveID) for each VM `Instance` object?

Reply:
Yea, I thought about that, but I figured since n is small, it may not be worth using a map. I wanted to write benchmarks to see if that is a bottle neck. If it turns out to be so, then we can switch to a `map[string]string`.